src/store/popularCategories/slice.ts

```diff
@@ -0,0 +1,46 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+import { getPopularCategories } from "api";
+import { Category, LOAD_STATUSES_TYPES, Good } from "types"
+
+const SLICE_NAME = "popularCategories";
+
+const popularCategoriesOnBack = createAsyncThunk(SLICE_NAME, getPopularCategories)
+
+
+export interface PopularCategoriesStore {
+       loadStatus: LOAD_STATUSES_TYPES;
+       data: {
+              category: Category,
+              items: Good[]
+       }[]
+}
+
+ const initialState: PopularCategoriesStore = {
+       loadStatus: LOAD_STATUSES_TYPES.SET_UNKNOWN,
+       data: [{
```

Comment:
для массивов начальное значение пустой массив, несуществующие мусорные данные добавлять не надо
```suggestion
       data: []
```